docroot/themes/custom/uids_base/scss/admin.scss

```diff
@@ -766,6 +766,12 @@ body.toolbar-vertical.toolbar-tray-open {
   }
 }
 
+// Hide certain system blocks.
+.block-extra-field-blocknodepagecontent-moderation-control,
+.block-extra-field-blocknodepagelinks {
+  @include element-invisible;
+}
+
```

Comment:
Can we add something like 
`.layout-builder-block--placeholder {
  &.block-extra-field-blocknodepagecontent-moderation-control,
  &.block-extra-field-blocknodepagelinks {
    @include element-invisible;
  }
}`
to hide it on the layout editor, but leave it visible when viewing draft pages.

Reply:
Yes! My mistake, I will test/push your suggestion